include/fuse_common.h

```diff
@@ -447,6 +452,18 @@ struct fuse_loop_config_v1 {
  */
 #define FUSE_CAP_DIRECT_IO_ALLOW_MMAP  (1 << 28)
 
+/**
+ * Indicates support for passthrough mode access for read/write operations.
+ *
+ * If this flag is set in the `capable` field of the `fuse_conn_info`
+ * structure, then the FUSE kernel module supports redirecting read/write
+ * operations to the backing file instead of letting them to be handled
+ * by the FUSE daemon.
+ *
+ * This feature is disabled by default.
+ */
+#define FUSE_CAP_PASSTHROUGH      (1 << 28)
```

Comment:
This conflicts with above FUSE_CAP_DIRECT_IO_ALLOW_MMAP

Reply:
I suppose this should simply be `1 << 29`

Reply:
yap. fixed. Thanks!